src/Extension/CoreExtension.php

```diff
@@ -2055,4 +2073,27 @@ public static function parseAttributeFunction(Parser $parser, Node $fakeNode, $a
 
         return new GetAttrExpression($args[0], $args[1], $args[2] ?? null, Template::ANY_CALL, $line);
     }
+
+    private static function getPropertyChecker(string $class, string $property): \Closure
+    {
+        static $classReflectors = [];
+
+        $class = $classReflectors[$class] ??= new \ReflectionClass($class);
+
+        if (!$class->hasProperty($property)) {
+            static $propertyExists;
+
+            return $propertyExists ??= property_exists(...);
```

Comment:
Twig 3 supports PHP 8.0, which doesn't support first-class callable syntax.
```suggestion
            return $propertyExists ??= static fn ($object, $property) => property_exists($object, $property);
```

Reply:
oops
updated using Closure::fromCallable